api/common/network.go

```diff
@@ -62,26 +59,26 @@ func GetObservedNetworkConfig(source corenetwork.ConfigSource) ([]params.Network
 	}
 	var namesOrder []string
 	nameToConfigs := make(map[string][]params.NetworkConfig)
-	sysClassNetPath := source.SysClassNetPath()
```

Comment:
Which means this need not be exposed.
The leaks in the abstraction are now plugged.